src/util.cc

```diff
@@ -76,6 +76,117 @@ parseOrReadJSONObject( const std::string & jsonOrPath )
 }
 
 
+/* -------------------------------------------------------------------------- */
+
+  std::vector<std::string>
+splitAttrPath( std::string_view path )
+{
+  std::vector<std::string> parts;
+
+  bool inSingleQuote = false;
+  bool inDoubleQuote = false;
+  bool wasEscaped    = false;
+  auto start         = path.begin();
+
+  /* Remove outer quotes and unescape. */
+  auto dequote = [&]( const std::string & part ) -> std::string
+    {
+      auto itr = part.begin();
+      auto end = part.end();
+
+      /* Remove outer quotes. */
+      if ( ( ( part.front() == '\'' ) && ( part.back() == '\'' ) ) ||
+           ( ( part.front() == '"' )  && ( part.back() == '"' ) )
+         )
+        {
+          ++itr;
+          --end;
+        }
+
+      /* Remove escape characters. */
+      std::string rsl;
+      bool        wasEscaped = false;
```

Comment:
```suggestion
      bool wasEscaped = false;
```

Reply:
I'm aligning the variable names and types.